packages/mermaid/src/defaultConfig.ts

```diff
@@ -253,6 +253,13 @@ const config: RequiredDeep<MermaidConfig> = {
     // TODO: can we make this default to `true` instead?
     useMaxWidth: false,
   },
+  packet: {
+    ...defaultConfigJson.packet,
+    useWidth: undefined,
```

Comment:
By the way, we only have this line in other diagrams' default config for backwards compatibility, just in case there is a difference between `{useWidth: undefined}` and `{}`.

If you're okay with `Object.keys()` not including `useWidth`, we can remove this line. After all, either way, `config.packet.useWidth` will still be `undefined`.